actors/builtin/reward/reward_logic.go

```diff
@@ -5,8 +5,30 @@ import (
 	big "github.com/filecoin-project/specs-actors/actors/abi/big"
 )
 
-var BaselinePowerAt = func(epoch abi.ChainEpoch) abi.StoragePower {
-	return big.NewInt(1 << 40) // PARAM_FINISH
+const BaselineExponentString = "340282663082994238536867392845056089438"
```

Comment:
Definition of the exponent would be great, just in case we need to recalculate it.

Reply:
we in fact do, iiuc, as the block time is changing back to 30 seconds 😂 here's how I calculated it: https://www.wolframalpha.com/input/?i=Round%5BExp%5BLog%5B1%2B50%25%5D%2F%28%281+tropical+year%29%2F%2825+seconds%29%29%5D*2%5E128%5D

Reply:
Yes, please specify the arithmetic here in a comment and/or link to a gist with code.

Reply:
Note that the number here is still using a 25 seconds epoch duration, we should update this to correspond to the 30 seconds block time when the changes land [340282722315436325019587904274019633912](https://www.wolframalpha.com/input/?i=Round%5BExp%5BLog%5B1%2B200%25%5D%2F%28%281+tropical+year%29%2F%2830+seconds%29%29%5D*2%5E128%5D) related to #748